tikv-configuration-file.md

```diff
@@ -883,6 +883,11 @@ Configuration items related to Raftstore.
 + Default value: `"9s"`
 + Minimum value: `0`
 
+### `right-derive-when-split`
+
++ When this configuration is set to `true`, the key of the original Region is reused for the Region starting from the maximum split key. Otherwise, the key is reused for the Region starting from the original Region's start key.
```

Comment:
```suggestion
+ Specifies the start key of the new Region when a Region is split. If this configuration item is set to `true`, the start key is the maximum split key. Otherwise, the start key is the original Region's start key.
```